core/appeal/service.go

```diff
@@ -217,6 +260,10 @@ func (s *Service) Create(appeals []*domain.Appeal) error {
 		return fmt.Errorf("inserting appeals into db: %w", err)
 	}
 
+	if err := s.auditLogger.Log(ctx, AuditKeyBulkInsert, appeals); err != nil {
+		s.logger.Error(fmt.Sprintf("failed to record audit log: %s", err))
```

Comment:
with `salt/log` we can do like this instead right?
```suggestion
		s.logger.Error("failed to record audit log","err", err)
```